### ISSUE TITLE
MediaFormat is null when SoundPlayer._onSystemAppResumed [BUG]

### ISSUE THREAD

**Issue by handofthecode** [bsutton/sounds#62]
## Sounds Version : 

- 1.0.0-beta.13

----------------------------------------------------------

## Severity

- Throws error

--------------------------------------------------------

## Platforms you faced the error 

- Iphone SE iOS 14.2

----------------------------------------------

## Stack trace

```
NoSuchMethodError: The getter 'name' was called on null.
Receiver: null
Tried calling: name

audio.dart in Audio.toString at line 314
Called from: string_patch.dart in _StringBase._interpolate
track.dart in Track.toString at line 49
Called from: string_patch.dart in _StringBase._interpolate
sound_player.dart in SoundPlayer._onSystemAppResumed at line 576
sound_player.dart in onSystemAppResumed at line 914
player_base_plugin.dart in PlayerBasePlugin.onSystemAppResumed.<fn> at line 282
base_plugin.dart in BasePlugin.forEachSlot at line 173
player_base_plugin.dart in PlayerBasePlugin.onSystemAppResumed at line 278
base_plugin.dart in BasePlugin.didChangeAppLifecycleState at line 72
binding.dart in WidgetsBinding.handleAppLifecycleStateChanged at line 704
binding.dart in ServicesBinding._handleLifecycleMessage at line 192
platform_channel.dart in BasicMessageChannel.setMessageHandler.<fn> at line 74
platform_channel.dart in BasicMessageChannel.setMessageHandler.<fn> at line 73
binding.dart in _DefaultBinaryMessenger.handlePlatformMessage at line 283

```

**Comment by bsutton** [comment]
Will try to look at this in the next week or two. I'm on leave and doing a diy house reno so I'm a little saturated at the moment.

**Comment by handofthecode** [comment]
Much appreciated. Good luck with the house.

**Comment by bsutton** [comment]
Fixed in 1.0.0 release.
